docs/tools/hedgehog-engine/rangers/files/material.md

```diff
@@ -0,0 +1,9 @@
+---
+description: Material tools for Sonic Frontiers
+---
+# Material Tools
+
+## HedgeEdit
+Editor tool, is used to edit material for frontiers.
+
+[Download :fontawesome-solid-download:](https://drive.google.com/file/d/1VDHTpPe1MNVlAYblLsDd3Oa1KpsLqLGs/view){ .md-button .md-button--primary}
```

Comment:
Please do not use Google Drive links for tools. Either use the assets system that's described in the README.md of HedgeDocs (it says it's for images, but it can work for every kind of file), or link directly to the tools' page.

Reply:
Update on this: HedgeEdit already exists as a tool in HedgeDocs now, so this page isn't needed anymore